packages/utils/telemetry-utils/src/telemetryTypes.ts

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryBaseLogger, TelemetryEventCategory } from "@fluidframework/common-definitions";
+
+/**
+ * Property types that can be logged.
+ *
+ * @remarks Logging entire objects is considered extremely dangerous from a telemetry point of view because people can
```

Comment:
I feel like I see this comment all over the place, it's a good point but I don't think it belongs here.

Reply:
removed comment